sql/psql/OMERO5.1__0/OMERO5.1DEV__20.sql

```diff
@@ -0,0 +1,106 @@
+-- Copyright (C) 2014 Glencoe Software, Inc. All rights reserved.
+-- Use is subject to license terms supplied in LICENSE.txt
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 2 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License along
+-- with this program; if not, write to the Free Software Foundation, Inc.,
+-- 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+--
+
+---
+--- OMERO5 development release upgrade from OMERO5.1DEV__20 to OMERO5.1__0.
+---
+
+BEGIN;
+
+CREATE OR REPLACE FUNCTION omero_assert_db_version(version varchar, patch int) RETURNS void AS '
+DECLARE
+    rec RECORD;
+BEGIN
+
+    SELECT INTO rec *
+           FROM dbpatch
+          WHERE id = ( SELECT id FROM dbpatch ORDER BY id DESC LIMIT 1 )
+            AND currentversion = version
+            AND currentpatch = patch;
+
+    IF NOT FOUND THEN
+        RAISE EXCEPTION ''ASSERTION ERROR: Wrong database version'';
+    END IF;
+
+END;' LANGUAGE plpgsql;
+
+SELECT omero_assert_db_version('OMERO5.1DEV', 20);
+DROP FUNCTION omero_assert_db_version(varchar, int);
+
+INSERT INTO dbpatch (currentVersion, currentPatch,   previousVersion,     previousPatch)
+             VALUES ('OMERO5.1',     0,              'OMERO5.1DEV',       20);
+
+--
+-- Actual upgrade
+--
+
+UPDATE pg_enum
+    SET enumlabel = 'kbar'
+    WHERE enumlabel = 'kBar' AND enumtypid = (
+        SELECT oid FROM pg_type WHERE typname = 'unitspressure'
+    );
```

Comment:
Fair enough as this corresponds to the development patches which are not meant to be used in production ugprade (and will likely be deleted anyways?).
Let's make sure we throughly check the production upgrade script works as expected wrt this `enumlabel`